packages/dds/sequence/src/intervalCollection.ts

```diff
@@ -171,6 +183,15 @@ export class SequenceInterval implements ISerializableInterval {
         return result;
     }
 
+    public getIntervalId(): string | undefined {
+        if (this.properties) {
+            const id = this.properties[reservedIntervalIdKey];
+            if (id !== undefined) {
```

Comment:
`if (id !== undefined && typeof id ==="string")` then you can remove the cast, which isn't safe, you are just telling type script you know better, but at runtime it could be anything